t/integration/app-routes.t

```diff
@@ -10,7 +10,7 @@ use Test::More;
 use YAML 'Load';
 use JSON 'decode_json';
 
-use lib ("$Bin/../../examples/sample-app/lib");
+use lib ("$Bin/../../lib", "$Bin/../../examples/sample-app/lib");
```

Comment:
Please remove `$Bin/../../lib` from the includes.
If you have any problems with running tests please use `prove -I./lib` or `perl -I./lib` and then run the tests.
Thanks